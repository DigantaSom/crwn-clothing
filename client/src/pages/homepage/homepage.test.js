import { shallow } from 'enzyme';

import HomePage from './homepage.component';

describe('HomePage component', () => {
  it('should render HomePage component', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
