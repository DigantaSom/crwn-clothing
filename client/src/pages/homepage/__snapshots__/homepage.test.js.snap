// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render HomePage component 1`] = `
<styled.div>
  <Profiler
    id="Directory"
    onRender={[Function]}
  >
    <Connect(Directory) />
  </Profiler>
</styled.div>
`;
